test/TestCase.php

```diff
@@ -26,18 +26,10 @@
 use common_persistence_Manager;
 use common_persistence_sql_dbal_Driver;
 use PHPUnit\Framework\TestCase as UnitTestCase;
-use oat\generis\test\MockObject;
+use PHPUnit\Framework\MockObject\MockObject;
```

Comment:
If you make this change, I think you can then remove the `oat\generis\test\PHPUnit_Framework_MockObject_MockObject` class.